src/agent/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+ADD_EXECUTABLE(RemoteDiscHealthMonitor "main.cpp" "HealthChecker.h" "HealthChecker.cpp")
```

Comment:
quotes are not required; also try to put files in separate lines as this list will grow